src/main/sensors/gyro.c

```diff
@@ -44,20 +45,29 @@ float gyroADCf[XYZ_AXIS_COUNT];
 static int32_t gyroZero[XYZ_AXIS_COUNT] = { 0, 0, 0 };
 static const gyroConfig_t *gyroConfig;
 static biquadFilter_t gyroFilter[XYZ_AXIS_COUNT];
+static biquadFilter_t gyroFilterNotch[XYZ_AXIS_COUNT];
+static biquadFilter_t gyroFilterOld[XYZ_AXIS_COUNT];
```

Comment:
I really don't like the use of the term 'old' in code - over time it becomes meaningless.
Can you change it so we have"
```
static biquadFilter_t gyroFilterNotch[XYZ_AXIS_COUNT];
static biquadFilter_t gyroFilterLpf[XYZ_AXIS_COUNT];
```